src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -39,18 +39,41 @@ String privacySettingButtonValue = (Boolean) request.getAttribute("isPrivate")?
     }
   </style>
 
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"> </script>
+
   <script>
+
+    //open add user box
+    $(document).ready(function(){
+      $("#adder").click(function() {
+        $("#addUserBox").show();
+      });
+    });
+
+    // close add user box when user clicks on x or anywhere outside of the box
+    $(document).ready(function() {
+      $(".close").click(function() {
+        $("#addUserBox").hide();
+      });
+    });
+
+    // show contents of drop down settings menu
+    function toggleSettingsDropdown(){
+      var dropDiv = document.getElementsByClassName("dropdown-content")[0];
+      if(dropDiv.style.display === "none"){
```

Comment:
If you're using jquery, I think there's a `.toggle` function that can replace these five lines.

Reply:
Oops yeah I forgot to change that.